Makefile

```diff
@@ -1,18 +1,29 @@
 RUSTUP = rustup
+
 TOOLCHAIN = $(shell cat rust-toolchain)
-CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+CARGO = $(RUSTUP) run $(TOOLCHAIN) --install cargo --color always
```

Comment:
📣  The original line was correct:
```
▶ rustup run --help
rustup-run 
Run a command with an environment configured for a given toolchain

USAGE:
    rustup run [FLAGS] <toolchain> <command>...
```

Reply:
True, I misread the help text of the command :)